photutils/isophote/isophote.py

```diff
@@ -726,10 +726,41 @@ def to_table(self):
         result : `~astropy.table.QTable`
             An astropy QTable with the main isophote parameters.
         """
-        return _isophote_list_to_table(self)
+        return _isophote_list_to_table(self, key_properties)
+
+    def get_names(self):
+        '''Print the names of the properties of an \
+                `~photutils.isophote.IsophoteList` instance.
+        '''
+        list_names = list(_get_properties(self).keys())
+        return list_names
 
 
-def _isophote_list_to_table(isophote_list):
+def _get_properties(isophote_list):
+    """
+    Return the properties of an \
+            `~photutils.isophote.IsophoteList` instance.
+
+    Parameters
+    ----------
+    isophote_list : `~photutils.isophote.IsophoteList` instance
+        A list of isophotes.
+
+    Returns
+    -------
+    result : `OrderedDict`
+        An OrderedDict with the list of the isophote_list properties
+    """
+    properties = OrderedDict()
```

Comment:
```suggestion
    properties = dict()
```

python 3.7+ `dict` now preserves order insertion.  This line is the reason for the merge conflict, but I had already changed this line to `dict()`.